Traceback (most recent call last):
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/simulate_stats_script.py", line 5, in <module>
    from stats.models import PlayerStat
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/stats/models.py", line 3, in <module>
    class PlayerStat(models.Model):
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/db/models/base.py", line 129, in __new__
    app_config = apps.get_containing_app_config(module)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/apps/registry.py", line 260, in get_containing_app_config
    self.check_apps_ready()
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/apps/registry.py", line 137, in check_apps_ready
    settings.INSTALLED_APPS
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/conf/__init__.py", line 81, in __getattr__
    self._setup(name)
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/conf/__init__.py", line 61, in _setup
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Requested setting INSTALLED_APPS, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
Traceback (most recent call last):
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/simulate_stats_script.py", line 5, in <module>
    from stats.models import PlayerStat
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/stats/models.py", line 3, in <module>
    class PlayerStat(models.Model):
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/db/models/base.py", line 129, in __new__
    app_config = apps.get_containing_app_config(module)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/apps/registry.py", line 260, in get_containing_app_config
    self.check_apps_ready()
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/apps/registry.py", line 137, in check_apps_ready
    settings.INSTALLED_APPS
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/conf/__init__.py", line 81, in __getattr__
    self._setup(name)
  File "/Users/borkoloco/Documents/Varios/projects/psh-game/backend/venv/lib/python3.11/site-packages/django/conf/__init__.py", line 61, in _setup
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Requested setting INSTALLED_APPS, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
New stat created!
New stat created!
New stat created!
New stat created!
New stat created!
New stat created!
New stat created!
